components/org.wso2.ppaas.manager.console/console/applications_form.jag

```diff
@@ -109,7 +109,7 @@ switch (formtype) {
 }
 
 if (dasStatsPublisherEnabled) {
-    meteringDashboardUrl = session.get('METERING_DASHBOARD_URL');
+    meteringDashboardUrl = application.get("meteringDashboardUrl");
```

Comment:
No. We are getting the meteringDashboardUrl from application object instead of getting it from session because now we are reading the dashboard urls from conf file at web app initialization and setting it to application object instead of session object.
monitoringDashboardUrl is retrieved from application object in menu_generator.jag